src/components/User/KycRequests/KycRequest.List.vue

```diff
@@ -1,146 +1,132 @@
-          >
-            <option :value="CHANGE_ROLE_TASKS.submitAutoVerification">
-              Submit auto verification
-            </option>
-            <option :value="CHANGE_ROLE_TASKS.completeAutoVerification">
-              Complete auto verification
-            </option>
-            <option :value="CHANGE_ROLE_TASKS.manualReviewRequired">
-              Manual review requried
-            </option>
-            <option value="">
-              Any
-            </option>
-          </select-field>
+        <select-field
+          class="app-list-filters__field"
+          v-model="filters.pendingTasks"
+          label="Pending tasks"
+        >
+          <option :value="CHANGE_ROLE_TASKS.submitAutoVerification">
+            Submit auto verification
+          </option>
+          <option :value="CHANGE_ROLE_TASKS.completeAutoVerification">
+            Complete auto verification
+          </option>
+          <option :value="CHANGE_ROLE_TASKS.manualReviewRequired">
+            Manual review requried
+          </option>
+          <option value="">
+            Any
+          </option>
+        </select-field>
 
-          <input-field
-            class="app-list-filters__field"
-            v-model.trim="filters.requestor"
-            label="Requestor"
-            autocomplete-type="email"
-          />
-        </div>
+        <input-field
+          class="app-list-filters__field"
+          v-model.trim="filters.requestor"
+          label="Requestor"
+          autocomplete-type="email"
+        />
       </div>
+    </div>
 
-      <div class="request-list__list-wrp">
-        <template v-if="list && list.length">
-          <div class="app-list">
-            <div class="app-list__header">
-              <span class="app-list__cell">
-                Email
-              </span>
-              <span class="app-list__cell app-list__cell--right">
-                State
-              </span>
-              <span class="app-list__cell app-list__cell--right">
-                Last updated
-              </span>
-              <span class="app-list__cell app-list__cell--right">
-                Role to set
-              </span>
-            </div>
-            <button
-              class="app-list__li"
-              v-for="item in list"
-              :key="item.id"
-              @click="toggleViewMode(item.requestor.id)"
-            >
-              <email-getter
-                class="app-list__cell app-list__cell--important"
-                :account-id="item.requestor.id"
-                is-titled
-              />
-              <span class="app-list__cell app-list__cell--right">
-                {{ item.state }}
-              </span>
-              <span class="app-list__cell app-list__cell--right">
-                {{ formatDate(item.updatedAt) }}
-              </span>
-              <span class="app-list__cell app-list__cell--right">
-                {{ item | deriveRoleToSet }}
```

Comment:
Filter implementation does not deleted and still exist

Reply:
Also, the component has unused `provide` [here](https://github.com/tokend/admin-panel/pull/75/files#diff-1a5055bc43cffbf3ac9408d0a740f396R163)

Reply:
`snakeToCamelCase` method is useless